### ISSUE TITLE
Debian package

### ISSUE THREAD

**Issue by chapmanjacobd** [theryangeary/choose#32]
Debian package would be nice. I use Fedora most of the time but sometimes need to connect to a Ubuntu server (ew)

**Comment by theryangeary** [comment]
Hi @chapmanjacobd, sorry for the delayed response.

**TLDR**, it's simpler to download the [packaging-agnostic, prebuilt binary](https://github.com/theryangeary/choose/releases/download/v1.3.1/choose), or [install from cargo](https://github.com/theryangeary/choose#installing-from-package-managers)

There's nothing wrong with using Debian or Ubuntu, they're great projects that power much of the world today (even if they may have some issues).

The process of getting a package into either of their repositories is rather involved. You can read about what it takes to become a Debian package maintainer here: https://www.debian.org/doc/manuals/maint-guide/start.en.html#socialdynamics

But there is good news! `choose` is small. It is only a binary. There is not even a manual page, because the help menu tells you everything there is to know. So any system you are working on that is running Debian or Ubuntu, you can just download the binary and load it in your path, no package or package manager necessary. If downloading from GitHub (whether via web interface or curl) is not your style, you can also [install from cargo](https://github.com/theryangeary/choose#installing-from-package-managers)
